datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -146,7 +146,7 @@ public SdxClusterResponse create(String name, @Valid SdxCustomClusterRequest cre
     }
 
     @Override
-    @CheckPermissionByAccount(action = AuthorizationResourceAction.CREATE_DATALAKE)
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.RESIZE_DATALAKE)
```

Comment:
why don't you check permission on resource level instead of account level? https://github.com/hortonworks/cloudbreak/tree/master/authorization-common#checkpermissionbyresourcename

Reply:
it wont work since there is no crn parameter for the method